src/App.tsx

```diff
@@ -1,61 +1,139 @@
 import './App.scss';
+import './assets/fonts/Cyberpunk/Cyberpunk.ttf';
+import './assets/fonts/BlenderProBook/BlenderProBook.otf';
+import { ChangeEvent, FormEvent, useState } from 'react';
+import { TodoList } from './components/TodoList';
+import { User } from './types/User';
+import { Todo } from './types/Todo';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+const getUser = (id: number): User | null => {
+  const foundUser = usersFromServer.find(user => user.id === id);
+
+  return foundUser || null;
+};
+
+export const todosWithUser: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
+
+const getId = (todos: Todo[]) => {
+  return Math.max(...todos.map(todo => todo.id)) + 1;
+};
 
 export const App = () => {
+  const [title, setTitle] = useState<string>('');
+  const [userId, setUserId] = useState<number>(0);
+  const [isTitleError, setIsTitleError] = useState<boolean>(false);
+  const [isUserError, setIsUserError] = useState<boolean>(false);
+  const [todos, setTodos] = useState<Todo[]>(todosWithUser);
+
+  const handleChangeTitle = (event: ChangeEvent<HTMLInputElement>) => {
+    setTitle(event.currentTarget.value
```

Comment:
```suggestion
    setTitle(event.target.value
```

https://stackoverflow.com/questions/10086427/what-is-the-exact-difference-between-currenttarget-property-and-target-property